dota2/features/lobby.py

```diff
@@ -0,0 +1,348 @@
+        Sends a message to the Game Coordinator requesting to create a lobby.
+
+        :param password: password of lobby
+        :type password: :class:`str`
+        :param options: options
+        :type options: :class:`dict`
+        """
+        return self.create_tournament_lobby(password=password, options=options)
+
+    def create_tournament_lobby(self, password="", tournament_game_id=None,
+                                tournament_id=0, options=None):
+        """
+        Sends a message to the Game Coordinator requesting to create a tournament lobby.
+
+        :param password: password of lobby
+        :type password: :class:`str`
+        :param tournament_game_id: tournament game id
+        :type tournament_game_id: :class:`int`
+        :param tournament_id: tournament id
+        :type tournament_id: :class:`int`
+        :param options: options
+        :type options: :class:`dict`
+        """
+        options = {} if options is None else options
+        options["pass_key"] = password
+
+        command = {
+            "lobby_details": options,
+            "pass_key": password
+        }
+        if tournament_game_id is not None:
+            command.update({
+                "tournament_game": True,
+                "tournament_game_id": tournament_game_id,
+                "tournament_id": tournament_id
+            })
+
+        if self.verbose_debug:
+            self._LOG.debug("Sending match CMsgPracticeLobbyCreate request")
+
+        self.send(EDOTAGCMsg.EMsgGCPracticeLobbyCreate, command)
+
+    def config_practice_lobby(self, lobby_id, options):
+        """
+        Change settings of the selected lobby.
+
+        :param lobby_id: target lobby
+        :type lobby_id: :class:`int`
+        :param options: options
+        :type options: :class:`dict`
+        """
+        options = {} if options is None else options
+
+        if self.verbose_debug:
+            self._LOG.debug("Changing lobby options of lobby %s", lobby_id)
+
+        self.send(EDOTAGCMsg.EMsgGCPracticeLobbySetDetails, options)
+
+    def request_practice_lobby_list(self):
+        """
+        Request a list of practice lobbies.
+        """
+        if self.verbose_debug:
+            self._LOG.debug("Requesting practice lobby list.")
+
+        jobid = self.send_job(EDOTAGCMsg.EMsgGCPracticeLobbyList, {})
+
+        @self.once(jobid)
+        def wrap_lobby_list(message):
+            self.emit(self.EVENT_LOBBY_LIST_ANSWER, message)
+
+        return jobid
+
+    def request_friend_practice_lobby_list(self):
+        """
+        Request a list of friend practice lobbies.
+        """
+        if self.verbose_debug:
+            self._LOG.debug("Requesting friend practice lobby list.")
+
+        jobid = self.send_job(EDOTAGCMsg.EMsgGCFriendPracticeLobbyListRequest, {})
+
+        @self.once(jobid)
+        def wrap_lobby_list(message):
+            self.emit(self.EVENT_LOBBY_FRIEND_LIST_ANSWER, message)
+
+        return jobid
```

Comment:
Same here. Just turn this into a `get_friend_practice_lobby_list` using `wait_msg` and return the response.